guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -163,7 +163,7 @@ endif::[]
 . Copy the generated `curl` command.
 . On the hosts that you want to register, run the `curl` command as `root`.
 
-ifdef::katello,satellite,orcharhino[]
+ifdef::katello,satellite[]
 [NOTE]
 ====
 For {RHEL} 6.3 hosts, the release version defaults to {RHEL} 6 Server and needs to be pointed to the 6.3 repository:
```

Comment:
I think by now RHEL 6.3 is unsupported, even from Red Hat's point of view. Only the last RHEL 6 version (6.10 IIRC) can still be supported. I wonder what the point of this note is and if we shouldn't drop it altogether.